test/puppetlabs/puppetdb/command/dlo_test.clj

```diff
@@ -41,12 +41,12 @@
         (is (empty? (compressible-files dir threshold))))))
 
   (testing "a directory with some old and new messages"
-    (let [dir (fs/temp-dir)
+    (let [dir (fs/temp-dir "dlo-compress-intro-old-and-new")
           threshold (-> 7 days)
           stale-timestamp (.getMillis (-> 8 days ago))]
-      (fs/touch (fs/file dir "foo") stale-timestamp)
-      (fs/touch (fs/file dir "bar") stale-timestamp)
-      (fs/touch (fs/file dir "baz"))
+      (doto (fs/file dir "foo") fs/create (#(fs/touch % stale-timestamp)))
+      (doto (fs/file dir "bar") fs/create (#(fs/touch % stale-timestamp)))
+      (fs/create (fs/file dir "baz"))
```

Comment:
Explained during standup, touch doesn't change the timestamp if its already created.